app/controllers/discovered_hosts_controller.rb

```diff
@@ -25,8 +26,19 @@ def index
 
   def show
     # filter graph time range
+    @general =    {:Highlights => [ "productname", "memorysize", "interface", "manufacturer", "architecture", "macaddress", "processorcount", "physicalprocessorcount", "discovery_boot"]}
+    @categories = {:Storage => ["blockdevice"],
+                   :Hardware => ["hardw", "manufacturer", "memo", "process"],
+                   :Software => ["bios", "os", "discovery"]}
```

Comment:
Wait, intstead of sane default, make empty value the default one and if there is no user defined one, use the pre-defined regexp. That's better for future upgrades.